images/encryption/keywrap/jwe/testingkeys.go

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
wondering if your testing keys files should end in `_test.go`to keep them from getting compiled in to runtime binary? Maybe Go compiler is smart enough to not include them since I assume they are unreferenced objects when linked, but I don't know enough about it.